anta/input_models/stun.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Module containing input models for services tests."""
+
+from __future__ import annotations
+
+from ipaddress import IPv4Address
+
+from pydantic import BaseModel, ConfigDict
+
+from anta.custom_types import Port
+
+
+class ClientAddress(BaseModel):
```

Comment:
Going though the test I think we should call this StunClientTranslation

Reply:
Updated test name as well.
Thanks!